### DISCUSSION TITLE
Cansado de ficar arrumando as const do seu código? AQUI A SOLUÇÃO

### DISCUSSION THREAD

**Discussion by DEVigodE** [Flutterando/forum#48] [Dicas]
### **Ta cansado de montar sua e tela e as mensagens de alerta aparecendo aos montes??**

**ANTES**
![Screenshot_4](https://user-images.githubusercontent.com/13280820/194432015-19acba32-78bc-4eb5-af33-8fd68086471f.png)

**DEPOIS -> CTRL+S**
![Screenshot_5](https://user-images.githubusercontent.com/13280820/194432016-170e8bae-8709-4000-ade4-e20477218371.png)

1. Abrir a configurações do seu VSCODE
![Screenshot_2](https://user-images.githubusercontent.com/13280820/194431715-e59944f5-524d-4f93-aa13-ec184c399c74.png)

2. Pesquisar por autofix, e colocar "always" e PRONTO.....
![Screenshot_3](https://user-images.githubusercontent.com/13280820/194431839-a4c08a6e-f438-4b4f-8871-31cec902c3f5.png)

**Comment by eduardoflorence** [comment]
Isto não funciona. Essa configuração é para selecionar todas as ocorrências de uma palavra que está buscando somente no trecho de código que está selecionado.

**Comment by nilopaim** [comment]
![image](https://user-images.githubusercontent.com/4059140/194649624-a331cae8-461f-4604-b4df-e83970daa3f3.png)

Coloca isso no settings.json. Funciona!

Fonte: [https://www.youtube.com/watch?v=znN1Eh_OVWA&ab_channel=Prof.DiegoAntunes](url)
